### PR TITLE
Cloud changelog - Sept 1st

### PR DESCRIPTION
**Pull request by amyblais**

### CODE REVIEW

File: source/install/cloud-changelog.md [hunk 1]
```diff
@@ -4,12 +4,34 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
...
+## Release 2022-09-01
+
+### Improvements
+
+#### User Interface (UI)
+ - Added most active Direct Message channels, new top Playbooks, least active channels, and new team members widgets to Insights.
+ - Introduced a new ``/marketplace command`` that brings up the marketplace modal for the Admin, and changed the ``/help`` command so that it now keeps the user internal to Mattermost.
+ - Team unreads are now calculated based on the channel membership and threads only. Team membership is no longer taken into account.
+ - For introducing Boards and Playbooks to new users, an “explore other tools in platform” item was added to the end user onboarding checklist.
+Added new API endpoint ``GET /api/v4/users/me/top/dms`` to get users' Direct Message channel insights.
```
Comment (cwarnermm) [inline source/install/cloud-changelog.md:23]:
```suggestion
- Added new API endpoint ``GET /api/v4/users/me/top/dms`` to get users' Direct Message channel insights.
```

File: source/install/cloud-changelog.md [hunk 2]
```diff
@@ -4,12 +4,34 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
...
+
+### Bug Fixes
+ - Fixed an issue for the customer portal where the completed Company Information screen read "Provide your company name and address".
+ - Fixed an issue where reading a thread on the mobile app caused a negative mention count to display on the web app.
+ - Fixed an issue where the user profile image persisted after user account deletion.
+
+### Known Issues
+ - The Playbooks left-hand sidebar does not update when a user is added to a run or playbook without a refresh.
+ - The runs and playbooks in the Playbooks left-hand sidebar does not have dot-menus that allow interaction with each item [MM-44752](https://mattermost.atlassian.net/browse/MM-44752).
```
Comment (cwarnermm) [inline source/install/cloud-changelog.md:32]:
```suggestion
 - The runs and playbooks in the Playbooks left-hand sidebar don't have dot-menus that allow interaction with each item [MM-44752](https://mattermost.atlassian.net/browse/MM-44752).
```

**Review by cwarnermm — COMMENTED** [review]
Feedback provided inline

**Review by dontoisme — APPROVED** [review]

File: source/install/cloud-changelog.md [hunk 3]
```diff
@@ -4,12 +4,36 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
...
+ - Added most active Direct Message channels, new top Playbooks, least active channels, and new team members widgets to Insights.
+ - Introduced a new ``/marketplace command`` that brings up the marketplace modal for the Admin, and changed the ``/help`` command so that it now keeps the user internal to Mattermost.
+ - Team unreads are now calculated based on the channel membership and threads only. Team membership is no longer taken into account.
+ - For introducing Boards and Playbooks to new users, an “explore other tools in platform” item was added to the end user onboarding checklist.
+ 
+#### API Changes
+ - Added a new API endpoint ``GET /api/v4/users/me/top/dms`` to get users' Direct Message channel insights.
+
+### Bug Fixes
+ - Fixed an issue for the customer portal where the completed Company Information screen read "Provide your company name and address".
```
Comment (marianunez) [inline source/install/cloud-changelog.md:28]:
```suggestion
 - Fixed an issue in the System Console > Subscription where the completed Company Information screen read "Provide your company name and address".
```

**Review by marianunez — APPROVED** [review]
Minor suggestion below

**Review by johndavidlugtu — APPROVED** [review]

File: source/install/cloud-changelog.md [hunk 4]
```diff
@@ -4,12 +4,41 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
...
 - [Release 2022-07-28](#release-2022-07-28)
-- [Release 2022-07-20](#release-2022-07-20)
+
+## Release 2022-09-01
+
+### Improvements
+
+#### Boards
+ - Added indexes to improve performance.
+ - Fixed a bug where the "new" button in Kanban columns didn't always work.
```
Comment (wuwinson) [inline source/install/cloud-changelog.md:20]:
```suggestion
 - Fixed a bug where the "new" button in Kanban columns didn't always work [issue-focalboard-3600](https://github.com/mattermost/focalboard/issues/3600).
```

File: source/install/cloud-changelog.md [hunk 5]
```diff
@@ -4,12 +4,41 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
...
+ - Fixed an issue where the user profile image persisted after user account deletion.
+
+### Known Issues
+ - The Playbooks left-hand sidebar does not update when a user is added to a run or playbook without a refresh.
+ - The runs and playbooks in the Playbooks left-hand sidebar don't have dot-menus that allow interaction with each item [MM-44752](https://mattermost.atlassian.net/browse/MM-44752).
+ - On the new Boards RHS from the channel Apps Bar, channel members who are not board admins are incorrectly able to see the **Unlink** board button. However, selecting the button doesn't actually unlink the board unless the user is a board admin [issue-focalboard-3600](https://github.com/mattermost/focalboard/issues/3600).
+ - On Boards, selecting the **+ New** button below a column on the Kanban view doesn't always create a new card. As a workaround, set a new default card template by going to the dropdown menu from the blue **New** button on the header of the board, opening the **Options** menu on any card template, and selecting **Set as default** [issue-focalboard-3676](https://github.com/mattermost/focalboard/issues/3676).
```
Comment (wuwinson) [inline source/install/cloud-changelog.md:41]:
```suggestion
```

**Review by wuwinson — CHANGES_REQUESTED** [review]
@amyblais Made a few tweaks. Everything else looks good!

**Review by wuwinson — APPROVED** [review]
LGTM!

File: source/install/cloud-changelog.md [hunk 6]
```diff
@@ -4,12 +4,40 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
...
 - [Release 2022-07-28](#release-2022-07-28)
-- [Release 2022-07-20](#release-2022-07-20)
+
+## Release 2022-09-01
+
+### Improvements
+
+#### Boards
+ - Added indexes to improve performance.
+ - Fixed a bug where the "new" button in Kanban columns didn't always work [issue-focalboard-3600](https://github.com/mattermost/focalboard/issues/3600).
```
Comment (cwarnermm) [inline source/install/cloud-changelog.md:20]:
```suggestion
 - Fixed a bug where the **New** button in Kanban columns didn't always work [issue-focalboard-3600](https://github.com/mattermost/focalboard/issues/3600).
```

**Review by cwarnermm — COMMENTED** [review]
Provided feedback inline.

File: source/install/cloud-changelog.md [hunk 7]
```diff
@@ -4,12 +4,40 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
...
+
+### Improvements
+
+#### Boards
+ - Added indexes to improve performance.
+ - Fixed a bug where the **New** button in Kanban columns didn't always work [issue-focalboard-3600](https://github.com/mattermost/focalboard/issues/3600).
+ - Fixed issues with 'single-user' mode.
+
+#### User Interface (UI)
+ - Added most active Direct Message channels, new top Playbooks, least active channels, and new team members widgets to Insights.
```
Comment (mkraft) [inline source/install/cloud-changelog.md:24]:
```suggestion
 - Added new Insights widgets: Most Active DMs, Top Playbooks, Least Active Channels, and New Team Members.
```

File: source/install/cloud-changelog.md [hunk 8]
```diff
@@ -4,12 +4,40 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
...
+ - Introduced a new ``/marketplace command`` that brings up the marketplace modal for the Admin, and changed the ``/help`` command so that it now keeps the user internal to Mattermost.
+ - Team unreads are now calculated based on the channel membership and threads only. Team membership is no longer taken into account.
+ - For introducing Boards and Playbooks to new users, an “explore other tools in platform” item was added to the end user onboarding checklist.
+ 
+#### API Changes
+ - Added a new API endpoint ``GET /api/v4/users/me/top/dms`` to get users' Direct Message channel insights.
```
Comment (mkraft) [inline source/install/cloud-changelog.md:30]:
```suggestion
 - Added new API endpoints:
    - ``GET /api/v4/users/me/top/dms``
    - ``GET /api/v4/users/me/top/threads``
    - ``GET /api/v4/teams/:team_id/top/team_members``
    - ``GET /api/v4/teams/:team_id:/top/threads``
```

**Review by mkraft — CHANGES_REQUESTED** [review]

**Review by mkraft — APPROVED** [review]

**Review by gabrieljackson — APPROVED** [review]

**Pull request merged**